model-api-gen-gradle-test/typescript-generation/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+import org.modelix.metamodel.gradle.GenerateMetaModelSources
+
+plugins {
+    base
+    alias(libs.plugins.node)
+    id("org.modelix.model-api-gen") apply false
+}
+
+node {
+    version.set("18.3.0")
+    npmVersion.set("8.11.0")
```

Comment:
We should probably use versions from the catalogue.

Reply:
This is not a Maven artifact or Gradle plugin. I don't think arbitrary properties can be defined in the catalog.

Reply:
Arbitrary versions can be defined in the catalog and be used. 
(I'm not sure, if it is intended, but it works.)

I actually did it in this PR in `model-api-gen-gradle-test/build.gradle.kts`.

```
subprojects {
    plugins.withType<NodePlugin> {
        project.extensions.configure<NodeExtension> {
            version.set(libs.versions.node)
            download.set(true)
        }
}
```

---

I removed the configuration here in `model-api-gen-gradle-test/typescript-generation/build.gradle.kts`, because the configuration in `model-api-gen-gradle-test/build.gradle.kts` is enough.